src/engine/source/router/router.hpp

```diff
@@ -46,8 +48,8 @@ struct Route
      * @param filter_function Filter events to send to environment
      * @param subscription Subscription to handle status
      */
-    Route(const std::string &name,
-          const std::string &environment,
+    Route(const std::string& name,
```

Comment:
Can we move the code out of this file and put it into a .cpp file?

Reply:
router is not used in this PR, is left as technical debt until API/threading definition.